models/responses.go

```diff
@@ -0,0 +1,29 @@
+package models
+
+type ErrorResponse struct {
+	Errors  []error           `json:"errors"`
+	Status  int               `json:"-"`
+	Headers map[string]string `json:"-"`
+}
+
+func NewErrorResponse(statusCode int, headers map[string]string, errors ...error) *ErrorResponse {
+	return &ErrorResponse{
+		Errors:  errors,
+		Status:  statusCode,
+		Headers: headers,
+	}
+}
+
+type SuccessResponse struct {
+	Body    []byte            `json:"-"`
+	Status  int               `json:"-"`
+	Headers map[string]string `json:"-"`
+}
```

Comment:
Just checking we are intentionally omitting these fields from the json?

Reply:
This is intentional; follows pattern from other api repos e.g. `dp-identity-api`